lib/block-supports/layout.php

```diff
@@ -25,6 +25,108 @@ function gutenberg_register_layout_support( $block_type ) {
 	}
 }
 
+function gutenberg_generate_common_flow_layout_styles() {
```

Comment:
Note: not all of the common styles have been added yet. In its current state, this PR is just a proof of concept that we can pass the rendered styles back up to the global styles stylesheet.